src/modules/Invoice/html_admin/mod_invoice_settings.html.twig

```diff
@@ -85,6 +85,12 @@
                         <input class="form-control" type="text" name="invoice_starting_number" value="{{ params.invoice_starting_number }}">
                     </div>
                 </div>
+                <div class="mb-3 row">
+                    <label class="col-md-3 col-form-label">{{ 'Invoice number zero padding width'|trans }}</label>
+                    <div class="col-md-6">
+                        <input class="form-control" type="number" name="invoice_number_zero_padding" value="{{ params.invoice_number_zero_padding|default(5) }}" min="1" max="20">
```

Comment:
Let's shorten the variable name down to `invoice_number_padding`

Reply:
Resolved with https://github.com/devadattas/FOSSBilling/commit/0bf8c921c9b174431ca305459a3776532af01102